cypress/e2e/project.spec.ts

```diff
@@ -105,7 +104,7 @@ describe("Project view", () => {
   });
 
   it("Branch selector", () => {
-    cy.get("div.commit div.stat.branch").click();
+    cy.get("div.commit div.stat.branch").click().click();
```

Comment:
Cypress uses js click events which behave differently than when a real user clicks. I don't fully understand why this doesn't work in Cypress, hence the workaround.

Would be nice if we could switch to Playwright, which doesn't have issues like this and has overall a better DX.